addons/mail/static/src/core/common/@types/models.d.ts

```diff
@@ -54,6 +56,7 @@ declare module "models" {
         "Persona": Persona,
         "res.country": Country,
         "res.groups": ResGroups,
+        "res.role": Role,
```

Comment:
Why is this `res.role`? The python model is `mail.role`, the JS model should be the same.